package.json

```diff
@@ -157,5 +157,13 @@
     },
     "resolutions": {
         "mojito-rb-gen/merge": "1.2.1"
+    },
+    "thirdparty-dependencies": {
```

Comment:
Is this a common pattern? Is it enforced by tooling or could be it be forgotten and drift when adding or modifying "out of band" libraries?